INSTALL.md

```diff
@@ -83,6 +87,10 @@ Assuming you have [homebrew](http://www.brew.sh) installed, you can install redi
 brew update
 brew install homebrew/versions/redis28
 ```
+Alternatively, you can sudo install the latest version of redis:
```

Comment:
@aliu104 In the past, we've tried to encourage organizations to use 2.8, since that is/was used in production. It might be confusing to the user to specify such an old version on the one hand (2.8) and immediately after give instructions on how to install the latest version. I would encourage you to test Qiita with the latest version of Redis and if it appears to work correctly, remove the instructions for 2.8.

Reply:
@charles-cowart That makes a lot of sense! We installed the latest version of Redis, and we have not ran into any problems with running the Qiita server yet but will update if that changes.